### DISCUSSION TITLE
Help Compiling - Conan

### DISCUSSION THREAD

**Discussion by ABagOfPotato** [Blinue/Magpie#20] [Q&A] [answered]
Hey!

I was wondering if you had any advice or could add compilation instructions? There's been a lot of hype around the 0.6 version coming up so a few people have been trying to compile it and have a look/mess around but so far no one has been able to get things working!

Most of the problems seem to be around the project using Conan as the package manager? I've tried myself and have enlisted the help of two friends who work as software developers but both have been unable to get the project compiling. I've also seen about 4 other people online having similar issues.

As far as I can tell all Conan seems to be doing is adding the JSON + fmt libraries? I'm not quite sure what to ask as I realise its not exactly something the developer should be spending their time doing, just thought I'd mention it as no one I know, professional or otherwise seems to be able to get the project working(or able to get Conan working nicely in VisualStudio, or have even used Conan anywhere else).

**Comment by Blinue** [comment, marked as answer]
You need [Conan Extension for Visual Studio](https://marketplace.visualstudio.com/items?itemName=conan-io.conan-vs-extension). I will write compilation instructions soon.
